frontend/src/components/common/form/Typeahead.tsx

```diff
@@ -70,7 +70,8 @@ export function TypeaheadField<T extends TypeaheadModel>({
     setFieldTouched(name, true);
     if (
       (!getIn(name, 'value') && val !== '' && getIn(values, name) === '') ||
-      getIn(values, name) === null
+      getIn(values, name) === null ||
```

Comment:
I think javascript falsy checks includes null, undefined and '', so you can probably remove this line

Reply:
I was wondering about that, I'll double check to confirm. Thanks!